README.rst

```diff
@@ -27,6 +27,14 @@ interface, can be stored in notebook metadata.
 Nbparameterise is written in Python 3, but it can handle notebooks that use
 Python 2.
 
+
+Additional Method of cell selection:
+
+Add a tag called "parameters" to the cell you want to use for the parameters
+Notice: Additionally tagged cells will be ignored
```

Comment:
Let's integrate this information into the main description, rather than adding a separate note. Something like:

> ... these are defined in the first code cell, or a cell with a 'parameters' tag, with regular...

Not sure what you mean about 'additionally tagged cells' ?

Reply:
> Thanks - I've gone through and identified a few specific changes to make.
> 
> Could you also look at adding a test for using a tagged cell rather than the first code cell?

I have added the test case to cover the parameter based notebook

Reply:
>> Not sure what you mean about 'additionally tagged cells' 
 I was referring to the fact it only takes the first occurrence of the parameters tag